.github/workflows/ci.yml

```
@@ -0,0 +1,37 @@
+name: GitHub CI
```

Has been named as following in other PRs 😅

```suggestion
name: CI workflow
```